fili-core/src/main/java/com/yahoo/bard/webservice/web/DataApiRequest.java

```diff
@@ -553,7 +559,7 @@ private static void validateTimeAlignment(
             return dimension.getDimensionFields();
         } else if (showFields.size() == 1 && showFields.contains(DimensionFieldSpecifierKeywords.NONE.toString())) {
             // Show no fields
-            return Collections.emptySet();
+            return new LinkedHashSet<>();
```

Comment:
I understand your comment now. You mean a singleton instance of an empty `LinkedHashMap`, not a map with one key-value pair, for which I have [the same concerns as with a singleton `LinkedHashSet`](https://github.com/yahoo/fili/pull/25#discussion_r77239307)